src/app/components/options-sidebar/time-clock/time-clock.component.css

```diff
@@ -1,10 +1,42 @@
 .content-ClockIn {
-  padding: 2.1rem 1rem;
+    padding: 2.1rem 1rem;
 }
 
 .timer {
-  align-items: center;
-  display: flex;
-  height: 100%;
-  justify-content: center;
+    align-items: center;
+    display: flex;
+    height: 100%;
+    justify-content: center;
 }
+
+.place-holder-danger::-webkit-input-placeholder {
+    /* WebKit, Blink, Edge */
+    color: #909;
+}
+
+.place-holder-danger:-moz-placeholder {
+    /* Mozilla Firefox 4 to 18 */
+    color: #909;
+    opacity: 1;
+}
+
+.placeholder-danger::-moz-placeholder {
+    /* Mozilla Firefox 19+ */
+    color: #DC3545 !important;
+    opacity: 1;
+}
+
+.placeholder-danger:-ms-input-placeholder {
+    /* Internet Explorer 10-11 */
+    color: #DC3545 !important;
+}
+
+.placeholder-danger::-ms-input-placeholder {
+    /* Microsoft Edge */
+    color: #DC3545 !important;
```

Comment:
why !important?
AFAIK, using this tag is discouraged.

Reply:
we can define a color palette and it will have a better organization and it will be easier to use.

Reply:
done